app.js

```diff
@@ -1,10 +1,28 @@
+// Get dependencies
```

Comment:
I am not exactly sure, the structure seemed to be in traditional js so I went ahead with it. Will migrate to es6 if required.